output/secrethub.i

```diff
@@ -19,6 +19,7 @@
 
 extern char *Read(char* path, char** errMessage);
 extern char *Resolve(char* path, char** errMessage);
+extern char *ResolveEnv(char** errMessage)
```

Comment:
You seem to be missing a semi colon `;` at the end 😉

Reply:
Solved